kong/plugins/bot-detection/handler.lua

```diff
@@ -0,0 +1,71 @@
+local BasePlugin = require "kong.plugins.base_plugin"
+local reg = require "rex_pcre"
```

Comment:
This shouldn't be used, it shouldn't even be a dependency of ours. Prefer the PCRE API available on `ngx`.